src/js/events/MetronomeActions.js

```diff
@@ -74,10 +65,21 @@ const MetronomeActions = {
               });
               resolve();
             } catch (error) {
-              handleError(error);
+              AppDispatcher.handleServerAction({
+                type: REQUEST_METRONOME_JOBS_ERROR,
+                data: error
+              });
+              reject();
             }
           },
-          error: handleError,
+          error(xhr) {
+            AppDispatcher.handleServerAction({
+              type: REQUEST_METRONOME_JOBS_ERROR,
+              data: xhr.message,
```

Comment:
@orlandohohmeier it seems like the work you're asking for Mat to complete is not part of the work he set out to do, nor part of this task, but it would be really nice to have this consistency throughout the codebase. Let's merge this in and complete the task you're referring to as a separate effort?
@MatApple please create a JIRA and then feel free to merge this in.